app/views/devise/registrations/_profile_form.html.erb

```diff
@@ -197,3 +197,14 @@
     </div>
   </div>
 </div>
+
+<script>
```

Comment:
Try again to do this in rails.

Even if you decide to keep it here this is not the right place, this should be in the webpacked base code.

Reply:
I've just found a [StackOverflow page](https://stackoverflow.com/questions/22626222/validate-that-at-least-one-check-box-is-selected-in-rails-non-habtm) that may help me (I don't know why it ddn't turn up in my search before!). I'll try using rails helpers and re-submit the PR.